projects/spacetimedb.com/package.yml

```diff
@@ -0,0 +1,37 @@
+distributable:
+  url: https://github.com/clockworklabs/SpacetimeDB/archive/refs/tags/v0.6.1-beta.tar.gz
+  strip-components: 1
+
+provides:
+  - bin/spacetime
+
+versions:
+  - 0.0.0
```

Comment:
This should probably be calver. Tea treats all calver as less than non-calver semver.